robolectric/src/main/resources/robolectric-deps.properties

```diff
@@ -0,0 +1,12 @@
+16=org.robolectric:android-all:4.1.2_r1-robolectric-r1
```

Comment:
This is a format change; old format was:
    org.robolectric\:android-all\:4.1.2_r1-robolectric-0=android-all-4.1.2_r1-robolectric-0.jar

We probably need an upgrade path for people...